static/panes/opt-pipeline.ts

```diff
@@ -40,19 +40,19 @@ import * as utils from '../utils.js';
 import {Toggles} from '../widgets/toggles.js';
 
 import {
-    LLVMOptPipelineBackendOptions,
-    LLVMOptPipelineOutput,
-    LLVMOptPipelineResults,
-} from '../../types/compilation/llvm-opt-pipeline-output.interfaces.js';
+    OptPipelineBackendOptions,
+    OptPipelineOutput,
+    OptPipelineResults,
+} from '../compilation/opt-pipeline-output.interfaces.js';
```

Comment:
Is this path change right?

Reply:
Huh, I agree it is a bit surprising at first glance... (I didn't edit this path manually, VS Code must have done it when I renamed the file.)

The app certainly does work as written: tests pass and I've manually checked this component appears as well.

The path doesn't actually match how you would navigate the files on disk, but somehow still works... Perhaps the `tsconfig.frontend.json` with multiple `rootDirs` (`["./static", "./types"]`) is somehow involved here.

In any case, you can see this somewhat odd path is already used by another type two lines down (`CompilationResult` on line 48). From searching other files in `static/panes`, many of them seem to use this path style, so I suppose it's okay to leave this as-is.

Reply:
Hmmm, I see. We should probably tweak it so the actual path can fit there, but I don't want to break things, it was hard enough to get typescript working, let alone sharing of files between front and backend.
You can keep it like this and we can spend some time on it some other time